fcrypt/fcrypt.go

```diff
@@ -125,30 +134,37 @@ type certificateChainInfo struct {
 
 // New create context for crypto operations.
 func New(
-	provider CertificateProvider, cryptocontext *cryptutils.CryptoContext,
+	provider CertificateProvider, cryptocontext *cryptutils.CryptoContext, serviceDiscoveryURL string,
 ) (handler *CryptoHandler, err error) {
-	handler = &CryptoHandler{certProvider: provider, cryptoContext: cryptocontext}
+	handler = &CryptoHandler{
+		certProvider:        provider,
+		cryptoContext:       cryptocontext,
+		serviceDiscoveryURL: serviceDiscoveryURL,
+	}
 
 	return handler, nil
 }
 
-// GetOrganization returns online certificate origanizarion names.
-func (handler *CryptoHandler) GetOrganization() (names []string, err error) {
-	certURLStr, _, err := handler.certProvider.GetCertificate(onlineCertificate, nil, "")
-	if err != nil {
-		return nil, aoserrors.Wrap(err)
+// GetServiceDiscoveryURLs returns service discovery URLs.
+func (handler *CryptoHandler) GetServiceDiscoveryURLs() (serviceDiscoveryURLs []string) {
+	certs, err := handler.getOnlineCert()
+	if err != nil || len(certs) == 0 {
+		log.Warningf("Can't get online certificate. Will be used default service discovery URL: %v", err)
```

Comment:
Display error here instead of warning.

Could we have warning in one place only:
```go
defer func() {
    if len(serviceDiscoveryURLs) == 0 {
        log.Warning("Service discovery URL can't be found in certificate and will be used from config")

        serviceDiscoveryURLs = append(serviceDiscoveryURLs, handler.serviceDiscoveryURL)
    }
} ()
```

Reply:
Done